.github/workflows/ci.yml

```diff
@@ -1,6 +1,6 @@
 name: Java CI
 
-on: [push, pull_request]
+on: [push]
```

Comment:
you arent suppose to touch this

Reply:
I don`t understend how I did that. I`m sure I did not even open that file. Is it possible that I changed that file by doing revert or other actions with git in terminal?